go/control/api/api.go

```diff
@@ -45,8 +45,9 @@ type NodeController interface {
 	// and shut down.
 	UpgradeBinary(ctx context.Context, descriptor *upgrade.Descriptor) error
 
-	// CancelUpgrade cancels a pending upgrade, unless it is already in progress.
-	CancelUpgrade(ctx context.Context) error
+	// CancelUpgrade cancels the specific pending upgrade, unless it is already in progress.
+	// TODO: currently pending upgrade name uniqueness is not enforced.
```

Comment:
Should we file an issue for this?

Reply:
Added a note to https://github.com/oasisprotocol/oasis-core/issues/3579 will do it as part of that.